### ISSUE TITLE
 gpio_sleep_set_direction

### ISSUE THREAD

**Issue by c08oliverbrian** [dragino/TrackerD#20]
ESP32 V1.0.4 gpio.h does not define  gpio_sleep_set_direction() used in common.cpp

**Comment by namejinnameyuan** [comment]
Yes, there is no statement, it can be used, I will upload the latest firmware, I changed the way to turn off GPIO, and successfully reduced the sleep current.
